ansible.cfg

```diff
@@ -0,0 +1,39 @@
+# config file for ansible -- http://ansible.com/
+# ==============================================
+#
+# Tuned config file for running freeipa playbooks.
+
+[defaults]
+# Set the log_path
+log_path = ~/freeipa-ansible.log
+forks = 20
+host_key_checking = False
+retry_files_enabled = False
+nocows = True
+#remote_user = root
+roles_path = roles/
+gathering = smart
+fact_caching = jsonfile
+fact_caching_connection = $HOME/ansible/facts
+fact_caching_timeout = 600
+callback_whitelist = profile_tasks
+inventory_ignore_extensions = secrets.py, .pyc, .cfg, .crt, .ini
+# work around privilege escalation timeouts in ansible:
+timeout = 30
+# Set the path for library & modules folder
```

Comment:
The library and also module_utils lines are the only adaptions that should be in this file. All other changes are not needed for ansible-freeipa.

Reply:
Well, all the lines are enhancements to make the playbook execution faster. I don't see  why we should not include them...

Reply:
I think we should not include them especially if enabled by default, because these lines will change behavior. In your use case these changes might be good to have, but in others they might have a bad impact.
Therefore, please only add the lines that are needed to get it working and no extra tuning.

Reply:
If you insist....I agree some of them might be a bit more "intrusive" like the ssh pipelining, but other are harmless, like retry_files_enabled = False. I will get ride f the more agressive ones, basically the fact caching and the ssh, I belive the rest are harmless and useful.